src/composable/chat/useChatHighlights.ts

```diff
@@ -222,28 +279,55 @@ export function useChatHighlights(ctx: ChannelContext) {
 		return toReactive(data.highlights);
 	}
 
+	function getAllIgnored(): Record<string, IgnoreDef> {
+		if (!data) return {};
+
+		return toReactive(data.ignores);
+	}
+
 	function updateId(oldId: string, newId: string): void {
 		if (!data) return;
+		if (useIgnored) {
+			const h = data.ignores[oldId];
+			if (!h) return;
 
-		const h = data.highlights[oldId];
-		if (!h) return;
+			data.ignores[newId] = h;
+			delete data.ignores[oldId];
 
-		data.highlights[newId] = h;
-		delete data.highlights[oldId];
+			h.id = newId;
+		} else {
+			const h = data.highlights[oldId];
+			if (!h) return;
 
-		h.id = newId;
+			data.highlights[newId] = h;
+			delete data.highlights[oldId];
+
+			h.id = newId;
+		}
```

Comment:
Determine higher up which store to use (`data.highlights`/`data.ignores`) based on the composable parameter, shouldn't be a need for an if/else here